### ISSUE TITLE
Post CI build logs in both success and failure cases

### ISSUE THREAD

**Issue by vjrantal** [thaliproject/Thali_CordovaPlugin#207]
Current success behavior:
* Fails to post https://github.com/thaliproject/TestDummy/pull/38#issuecomment-150186016

Current failure behavior:
* ThaliTester posts something as an "inline comment" https://github.com/thaliproject/TestDummy/pull/38#issuecomment-150181679

Expected behavior would be to post the full logs onto a branch similarly in both cases.

**Comment by vjrantal** [comment]
I tested this after it was closed, but both the success and failure behavior are still as in the issue description so reopening.

**Comment by vjrantal** [comment]
@obastemur: I noticed one thing that could be improved in the way @ThaliTester reports the results. Currently, if you follow the link, for example, on this comment https://github.com/thaliproject/Thali_CordovaPlugin/pull/216#issuecomment-151153480 you will not end up to the results of building based on that commit. That is, because the PR later on triggered other builds that created new commits to the same branch - see https://github.com/ThaliTester/TestResults/commits/485472919331f37_Merge_story_001_vjrantal_to_story_001_yarong_vjrantal.

It would be important that the comments have a permalink to the right results, because otherwise, this will create confusion among people unaware of how exactly this works.

**Comment by obastemur** [comment]
Thats not how it really works. Did check what is exactly inside the other commit? 

Simply the result branch is not per PR, its per commit.

**Comment by vjrantal** [comment]
I posted the comment above by comparing https://github.com/thaliproject/Thali_CordovaPlugin/pull/216#issuecomment-151153480 and https://github.com/thaliproject/Thali_CordovaPlugin/pull/216#issuecomment-151159827 which both have the same URL.

However, now that I read your comment above and looked more carefully, the issue was that the CI re-triggered the build, because the PR assignment was changed (the commit wasn't touched). That is why the build results went into the same branch.

So I think the bug here is that CI should not re-trigger unless the commit actually changes?
